### PR TITLE
feat(cxl-ui): add cxl-playbook-progress-bar

### PR DESCRIPTION
**Pull request by saas786**
related issue: https://github.com/conversionxl/aybolit/issues/131

### CODE REVIEW

File: packages/storybook/cxl-ui/cxl-app-layout/layout=2c-l.stories.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const CXLAppLayout2cl = () => {
             <span class="progress statement playbook-completion-rate"
               >Completed 4 steps of 8 in total</span
             >
-            <vaadin-progress-bar value="0.50">Completed 4 steps of 8 in total</vaadin-progress-bar>
+            <cxl-playbook-progress-bar></cxl-playbook-progress-bar>
```
Comment (lkraav) [inline packages/storybook/cxl-ui/cxl-app-layout/layout=2c-l.stories.js:113]: Bring back progress statement text node.

File: packages/cxl-ui/src/components/cxl-progress-bar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { customElement } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import { ProgressBarElement } from '@vaadin/vaadin-progress-bar/src/vaadin-progress-bar';
+
+@customElement('cxl-playbook-progress-bar')
```
Comment (lkraav) [inline packages/cxl-ui/src/components/cxl-progress-bar.js:5]: Filename is still wrong.

File: packages/cxl-ui/src/components/cxl-progress-bar.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+export class CXLPlaybookProgressBar extends ProgressBarElement {
+  ready() {
+    super.ready();
+
+    document.addEventListener('cxl-playbook-accordion-changed', async (event) => {
```
Comment (lkraav) [inline packages/cxl-ui/src/components/cxl-progress-bar.js:10]: Possible to narrow down listening scope, ideally to specific accordion ID?

**Review by lkraav — CHANGES_REQUESTED** [review]

File: packages/cxl-ui/src/components/cxl-progress-bar.js (hunk 3 above)
Comment (saas786) [reply packages/cxl-ui/src/components/cxl-progress-bar.js]: Updated, please take a look now.

**Pull request merged**